packages/design-preview/src/components/sidebar.tsx

```diff
@@ -1,14 +1,22 @@
 import { translate } from 'i18n-calypso';
 import StyleVariationPreviews from './style-variation';
-import type { StyleVariation } from '@automattic/design-picker';
+import type { StyleVariation } from '@automattic/design-picker/src/types';
 
 interface SidebarProps {
 	title?: string;
 	description?: string;
 	variations?: StyleVariation[];
+	activeVariation?: StyleVariation;
```

Comment:
`variations` and `onVariationClick` is always defined when passed, can we make them required for clarity?

Reply:
Will do, updated 🙂